pynestml/codegeneration/printers/type_symbol_printer.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# type_symbol_printer.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+import abc
+
+from pynestml.symbols.symbol import Symbol
+from pynestml.codegeneration.printers.symbol_printer import SymbolPrinter
+
+
+class TypeSymbolPrinter(SymbolPrinter, metaclass=abc.ABCMeta):
```

Comment:
Why does `TypeSymbolPrinter` need to inherit from `SymbolPrinter`? Since both have the same `print` function and the `SymbolPrinter` is not used anywhere else, can we eliminate it altogether?

Reply:
You are correct, but I'd like to keep this organization, because in principle we might also be printing other symbols (like ``FunctionSymbol``s).